kbase-extension/static/kbase/js/common/cellComponents/tabs/results/resultsTab.js

```diff
@@ -134,6 +134,7 @@ define(['bluebird', 'common/ui', 'common/events', './outputWidget', './reportWid
                         const ref = objectKeys[idx];
                         createdObjects[ref].name = info[1];
                         createdObjects[ref].type = info[2];
+                        createdObjects[ref].wsInfo = info;
```

Comment:
The whole workspace info is needed for starting up a viewer cell - this info gets passed along to the list of created objects.